src/index.ts

```diff
@@ -0,0 +1,7 @@
+export function sampleFunction(message: string): void {
+  console.log(`${message}`);
+  //console.log("test");
+}
+
+let userMessage = "a message for the console";
```

Comment:
```suggestion
const userMessage = "a message for the console";
```
You don't change the value of the variable, so `const` is more appropriate.